examples/kubernetes-ingress/scripts/02-install-kubernetes-master.sh

```diff
@@ -63,7 +63,7 @@ Documentation=https://kubernetes.io/docs/home
 
 [Service]
 ExecStart=/usr/bin/kube-apiserver \\
-  --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,NodeRestriction,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds \\
+  --enable-admission-plugins=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeClaimResize,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,Priority \\
```

Comment:
Where do these defaults come from? Would be useful to link to documentation that specifies that these are the defaults.